### PR TITLE
Add bounds argument

### PR DESCRIPTION
**Pull request by rjimenezda**
This PR adds a bounds argument, if it's present they won't be automatically calculated.

The format is a dict with `west`, `east`, `north`, `south` for convenience. Maybe it's better to provide a bounding box array instead? I always screw those up though :joy:

### CODE REVIEW

File: cartoframes/contrib/vector.py [hunk 1]
```diff
@@ -326,12 +330,17 @@ def vmap(layers, context, size=(1024, 632), basemap=BaseMaps.voyager):
...
+        size (integer tuple): size in pixels of the generated iframe with the
+          map.
         basemap (str):
           - if a `str`, name of a CARTO vector basemap. One of `positron`,
             `voyager`, or `darkmatter` from the :obj:`BaseMaps` class
           - if a `dict`, Mapbox or other style as the value of the `style` key.
             If a Mapbox style, the access token is the value of the `token`
             key.
+        bounds (dict): a dict with `east`/`north`/`west`/`south` properties.
+          If not provided the bounds will be automatically calculated to fit all
```
[the remark this answers is not in the document]
Comment (rjimenezda) [reply cartoframes/contrib/vector.py]: Oh my god I'm going to commit puppycide

**Comment by rjimenezda** [comment]
Sorry about the hound noise, my editor was not properly configured. We should probably squash this PR :joy:

**Comment by andy-esch** [comment]
Looks great 👍

Besides the comments I added, I was under the impression that we were going to be adding zoom and center like in `CartoContext.map(..., zoom=None, lng=None, lat=None, ...)`. If that's the case we need to give preference to bounds over zoom/center (or opposite) if a user enters both. And there'd need to be additional logic in the vector.{html,py} files. But maybe that's for another PR / another day.

I just re-started the tests. It looks like there were some tables left over in the testing account from a bad run.

File: cartoframes/contrib/vector.py [hunk 2]
```diff
@@ -326,12 +330,17 @@ def vmap(layers, context, size=(1024, 632), basemap=BaseMaps.voyager):
         context (:py:class:`CartoContext <cartoframes.context.CartoContext>`):
           A :py:class:`CartoContext <cartoframes.context.CartoContext>`
           instance
+        size (integer tuple): size in pixels of the generated iframe with the
```
Comment (andy-esch) [inline cartoframes/contrib/vector.py]: Great catch! 

How about `size (tuple of ints)` instead?

And for the description, maybe something that leads them to understand how to structure the size: `A (width, height) pair for the size of the map. Default is '(1024, 632)'.`

**Comment by rjimenezda** [comment]
To be honest, I added this on a whim cause I wanted to check out linting, PRs etc, I didn't really stop to read what we were going for.

I'll see how hard it is to add it as well 👀

**Comment by andy-esch** [comment]
No worries :) I've found the cat comments to be really funny!

**Comment by rjimenezda** [comment]
Updated to support a list, plus better docs and :sparkles: tests :sparkles: 

I think we can tackle the center / zoom on a separate one.

**Comment by andy-esch** [comment]
Beautiful :)

**Pull request merged**